openreview/conference/builder.py

```diff
@@ -1533,6 +1536,35 @@ def is_release_authors(is_note_accepted):
             self.set_homepage_decisions(decision_heading_map=decision_heading_map)
         self.client.remove_members_from_group('active_venues', self.id)
 
+    def post_decisions(self, decisions_file):
+        decisions_data = list(csv.reader(StringIO(decisions_file.decode()), delimiter=","))
+
+        def post_decision(paper_decision):
+            try:
+                paper_id, decision, comment = paper_decision
+            except IndexError:
```

Comment:
what is an IndexError?

Reply:
It should be `ValueError`, but I did this to keep the `comment` field optional, so if the decisions csv only have paperIDs and decisions it will run without any error.

Reply:
When is it happening the `ValueError`?  AFAIK `paper_decision` is a tuple with three values and you are assign them to three different variable, right?

Reply:
I believe since the comment if optional, then sometimes paper_decision will only have two values which we will try to assign to three variables

Reply:
I get this error when only passing paper_id and decision:

`ValueError('not enough values to unpack (expected 3, got 2)')`

Reply:
Yes, I'll update it to `ValueError` in place of `IndexError`.

Reply:
ohhhh I see, it can be simpler to treat `paper_decision` as an array. We need to make sure that `paper_id` and `decision` are always present. So the array could be length 2 or 3.

Reply:
Also, might be a good idea to strip() the decisions? If an extra space is added before or after, it will not match the invitation definition